### ISSUE TITLE
Running npm install error

### ISSUE THREAD

**Issue by svallory** [trabian/titanium-backbone#4]
Running 

`$ sudo npm install -g`

I got the following error:

https://gist.github.com/2819925.js

**Comment by svallory** [comment]
The problem is actually with the stitch-up which defines the path to a personal repository on the stitch dependency.

http://registry.npmjs.org/stitch-up

**Comment by trabianmatt** [comment]
@svallory, Thanks -- I was mid-fix when I got your note. This project is still quite immature but it represents an in-process extraction of patterns and helpers from a fairly large Titanium app currently in production, so please let me know your thoughts.

Thanks!

**Comment by svallory** [comment]
Well, I'm researching js frameworks compatible with Titanium. kranium and t-b appealed more to me. As kranium seems to be a bunch of stuff glued together (didn't try it yet) I thought about t-b yours first. I'll give it a try, but I'm having a hard time installing it. Just now I got `env: coffee: No such file or directory` and I had to manually install coffee-script. I think you missed that dependancy. I'll get back to you after I've tried it.
